src/uwtools/tests/config/test_support.py

```diff
@@ -12,13 +13,21 @@
 from uwtools.config.formats.fieldtable import FieldTableConfig
 from uwtools.config.formats.ini import INIConfig
 from uwtools.config.formats.nml import NMLConfig
+from uwtools.config.formats.sh import SHConfig
 from uwtools.config.formats.yaml import YAMLConfig
 from uwtools.exceptions import UWConfigError
 from uwtools.logging import log
 from uwtools.tests.support import logged
 from uwtools.utils.file import FORMAT
 
 
+def test_config_sections():
+    cfg = configparser.ConfigParser()
+    sections = support.config_sections(cfg)
+    cfg.read_string("[top]\n")
+    assert not dict(sections.get("top"))
```

Comment:
Could we do something here that demonstrates a little better what the code-under-test is providing? This seems like kind of a degenerate case. We could e.g.
```
(Pdb) !cfg.read_string("[flower]\nroses=red\n[fruit]\na=apple\n")
(Pdb) p sections
{'flower': {'roses': 'red'}, 'fruit': {'a': 'apple'}}
```
and make some assertions about that.

Reply:
Ah, thats true - I left this as a blunt version of the format test, but it'll certainly be more meaningful with some detail

Reply:
100% coverage is necessary but not sufficient :)